cumulusci.yml

```diff
@@ -13,19 +13,23 @@ tasks:
         description: Deploys the post install configuration for an unmanaged DE org
         class_path: cumulusci.tasks.salesforce.Deploy
         options:
-            path: dev_config/src
+            path: unpackaged/config/dev/src
```

Comment:
@davidmreed For NPSP the metadata is directly inside unpackaged/config/dev without a src folder. Is there a compelling reason to add the src folder? I don't have a huge preference, but it seems nice to avoid an extra layer of depth, and I think we should be consistent between projects.

Reply:
No, there's not a compelling reason - I will remove that extra layer of hierarchy.